release-notes/System.ServiceModel.Duplex.md

```diff
@@ -0,0 +1,15 @@
+# System.ServiceModel.Duplex
+
+## Information
+
+* [NuGet Package](https://nuget.org/packages/System.ServiceModel.Duplex)
+* [Report an issue](http://github.com/dotnet/wcf/issues/new)
+* [Source](https://github.com/dotnet/wcf/tree/master/src)
```

Comment:
I duplicated these pages from the Core repo and tweaked them for our links.  Obviously we can customize them to fit our needs, but that is why they look like they do.